docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -461,12 +458,40 @@ is external so it's handy for Getting Started to share -->
         <title>Auto Scale core concepts</title>
         <para>To use the &PROD; API effectively, you should understand
             several key concepts: </para>
-        <section xml:id="Glossary">
-            <title>Glossary</title>
-            <!-- Glossary section, defining basic terms,
-is external so it's handy for Getting Started to share -->
-            <xi:include href="chapters/glossary.xml" xpointer="element(Glossary-000)"/>
-            <!-- -->
+        <section xml:id="How_Auto_Scale_Works">
+            <title>How Auto Scale Works</title>
+            <para>Rackspace &PROD; is an API-based tool that automatically scales resources in response to
+                an increase or decrease in overall workload based on user-defined thresholds.</para>
+            <para>&PROD; functions by linking these services:
+                <itemizedlist>
+                    <listitem>
+                        <para>&PROD; API, described in this guide.</para>
+                    </listitem>
+                    <listitem>
+                        <para>Load-balanced Cloud Servers. For more information, read the Rackspace
+                            <link xlink:href="http://docs.rackspace.com/loadbalancers/api/v1.0/clb-getting-started/content/DB_Doc_Change_History.html">Load Balancers Getting Started Guide
+                            </link>.</para>
+                    </listitem>
+                </itemizedlist>You can set up a schedule for launching Auto Scale or define an event that triggers a
+                webhook. You can also specify a minimum and maximum number of cloud servers for your
+                scaling group, the amount of resources you want to increase or decrease, and policies
+                based on percentage or real numbers.</para>
+            <para>
+                <note>
+                    <para>&PROD; does not configure any information within a server. You must configure your
+                        services to function when each server is started. We recommend automating your
+                        servers' startup processes with
+                        <link xlink:href="http://www.opscode.com/chef/">Chef</link> or a similar tool.</para>
+                </note>
+            </para>
+            <para>
+                <note>
+                    <para>&PROD; can utilize all create and delete server Rackspace Cloud Server API operations. For more details, 
```

Comment:
Yeah, I would go with "API parameters" or "configurations" or something, not "operations" for this case.